releases/net471/KnownIssues/521954 - BCL CultureAwareComparer with ignore casing on serialized on previous versions of .NET do not correctly deserialize on .NET 4.7.1.md

```diff
@@ -21,4 +21,8 @@ P:System.StringComparer.CurrentCultureIgnoreCase
 
 Ensure that both serialization and deserialization occurs on systems running a version of the .NET Framework starting with 4.7.1.
 
-We are actively working on an update to address this issue and we will update this document with a link to it once it becomes available.
+If you are running on Windows Fall Creators Update you can use the following links to install an update that will address this issue. Please select the corresponding link based on the processor architecture:
```

Comment:
@AlexGhiondea, did you still want this PR reviewed? (I'm not sure why you closed it.) If so, the text is somewhat redundant ("following links" and "corresponding links"). It can be shortened to:
If you are running on WIndows Fall Creators Update, an update is available that addresses this issue. Select one of the following links based on your processor architecture: